### PR TITLE
BigQuery copy materialization description

### PR DESCRIPTION
**Pull request by xemuliam**
## Description & motivation
BigQuery copy materialization description

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [X] Yes

### CODE REVIEW

**Comment by annafil** [comment]
Thank you this is a really great contribution! I would like to take a pass at this with you on copyediting this a bit, but after that we should merge it in!

**Comment by xemuliam** [comment]
@annafil , @jtcohen6 
do we have any update please?

**Comment by xemuliam** [comment]
I've merged all latest changes from NEXT branch.
No conflicts there at the moment.
Do we have any change to get this change merged?

File: website/docs/docs/building-a-dbt-project/building-models/materializations.md [hunk 1]
```diff
@@ -100,3 +104,24 @@ When using the `table` materialization, your model is rebuilt as a <Term id="tab
     * very light-weight transformations that are early on in your DAG
     * are only used in one or two downstream models, and
     * do not need to be queried directly
+
+## Copy (BigQuery only)
+<Changelog>New in v0.21.0</Changelog>
```
Comment (runleonarun) [inline website/docs/docs/building-a-dbt-project/building-models/materializations.md]: I think we can remove these now that we've deprecated versions before 1.0.
```suggestion
```

File: website/docs/reference/resource-configs/bigquery-configs.md [hunk 2]
```diff
@@ -636,3 +636,91 @@ access to `project_1.dataset_1` and `project_2.dataset_2`.
 
 #### Limitations
 The `grant_access_to` config is not thread-safe when multiple views need to be authorized for the same dataset. The initial `dbt run` operation after a new `grant_access_to` config is added should therefore be executed in a single thread. Subsequent runs using the same configuration will not attempt to re-apply existing access grants, and can make use of multiple threads.
+
+## Copy materialization
+<Changelog>New in v0.21.0</Changelog>
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/bigquery-configs.md]:
```suggestion
```

**Comment by runleonarun** [comment]
@xemuliam Once we get a dbt technical review, the docs team can help get this PR merged. We appreciate your contributions and are trying to make the process both quicker and smoother.

@gwenwindflower Being that you just wrote the [guide on Materializations](https://docs.getdbt.com/guides/best-practices/materializations/2-available-materializations), can you please give this PR a technical review so we can get it merged? 🙇🏻

**Pull request closed without merging**